examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -411,10 +417,37 @@ def act(self, obs, env):
         human_pos = env._sim.agents_mgr[
             1 - self._agent_idx
         ].articulated_agent.base_transformation.translation
+        human_rot = env._sim.agents_mgr[
+            1 - self._agent_idx
+        ].articulated_agent.base_rot
         robot_pos = env._sim.agents_mgr[
             self._agent_idx
         ].articulated_agent.base_transformation.translation
 
+        # Check if the human walks or rotates
+        is_human_walk = (
+            np.linalg.norm(
+                np.array((self._prev_human_pos - human_pos))[[0, 2]]
+            )
+            > 0.01
+            if self._prev_human_pos is not None
+            else False
+        )
+        is_human_rotate = (
+            (
+                (np.abs(float(self._prev_human_rot - human_rot)) + np.pi)
+                % (2.0 * np.pi)
+                - np.pi
+            )
+            > 0.01
```

Comment:
Same here.

Reply:
Thank you, Eric! I added `DISTANCE_ROTATION_TRESHOLD_HUMAN_MOVING` for this.